apps/app/src/views/routes/SelectOrg.tsx

```diff
@@ -43,7 +43,8 @@ export const SelectOrg = () => {
         login({
           appState: {
             returnTo: from
-          }
+          },
+          organizationId: organizations[0].id
```

Comment:
do we need to optionally chain this?

Reply:
I dont think so, because `organizations` has length === 1